### DISCUSSION TITLE
2022-05-11 After HARMONY discussion

### DISCUSSION THREAD

**Discussion by Gonza10V** [SynBioDex/SBOL-Editors#2] [Editors meetings]
- We assign people to non-sep issues in https://github.com/SynBioDex/SBOL-specification
- How to manage SBOL examples and best practices, we decide to try GitHub Discussions to link Issues and SEP including the community in the discussion and transparent the process about how we reach that consensus.

Proposed workflow from [SBOL-specification#478](https://github.com/SynBioDex/SBOL-specification/issues/478)
1. For a potential best practice example, open a discussion topic on the sbol-editors repository.
2. If an issue exists on this topic, link to it to see the past discussion, but have all future discussion on the discussion thread.
3. When a solution is being formulated, create a markdown description following the SEP format. Where appropriate, there should be fully formulated SBOL example(s).
4. When the solution appears to have achieved consensus (determined by the SBOL editors), then a link to this solution is added to the FAQ on the SBOL standard website.
5. Note that not all discussion topics need to ever be promoted to best practices linked on the FAQ.

**Comment by cjmyers** [comment]
Markdown and examples should be prepared by the community.

SBOL editors can decide to determine consensus via community vote when there are objections during the finalization period.

Note that the data model should never be changed by the example (ontology terms are not changes to the data model in all cases).

**Comment by cjmyers** [comment]
Possibly move this to a new best practices repository

**Comment by cjmyers** [comment]
Rename this to SBOL Community Discussions or something similar

**Comment by Gonza10V** [comment]
I think this repo is a way to interact with the community, part of it can become a “best practice” or “recommendation” or even an “example”. We can decide where that stuff lives later, it can also become Python code and its place can be decided at the end of the discussion itself.
We can also make a poll to decide the name, you can toss here your ideas.
